pkg/piperutils/cyclonedxBom.go

```diff
@@ -46,3 +48,22 @@ func GetBom(absoluteBomPath string) (Bom, error) {
 	}
 	return bom, nil
 }
+
+func GetPurl(filePath, bomFilename string) string {
```

Comment:
Keep it simple, directly send the bomFilePath and get the purl. With this you don't need to make much change in maven and npm